.env.example

```
@@ -0,0 +1,39 @@
+# --- Required ---
+# Prisma/Postgres connection string (replace password-add-here with your password)
+DATABASE_URL=postgresql://postgres:password-add-here@localhost:5432/helprr
+
+# Redis connection string (required for login rate limiting)
+REDIS_URL="redis://helprr-redis:6379"
+
+# --- Optional: server-side media cache ---
+# Directory used for image byte cache
+IMAGE_CACHE_DIR=/tmp/helprr-image-cache
+# Freshness window for cached images (seconds)
+IMAGE_CACHE_TTL_SECONDS=604800 // 7 days
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)